### PR TITLE
add(epp): Alyvix Server monitoring procedure

### PR DESCRIPTION
**Pull request by thibaults-centreon**
## Target serie

- [X] 20.04.x
- [X] 20.10.x (master)

### CODE REVIEW

File: en/integrations/plugin-packs/procedures/applications-monitoring-alyvix-restapi.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+Build end-user bots visually interacting with any Windows application like ERPs or your favourite browser. 
+Measure end-user experiences: Alyvix records the click-to-appearance responsiveness of each transaction.
+Report IT service quality to support technical and business actions.
+
+The Centreon Plugin-Pack *Alyvix Server* aims to collect the execution status and duration of the Alyvix's *testcases* and their *transactions*
+by requesting the dedicated built-in RestAPI.
+
+> The *Alyvix Server* Plugin-Pack **can not** be used with the Open-Source free version of Alyvix, as the Rest API feature is only
+> available in the commercial version.
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/applications-monitoring-alyvix-restapi.md:18]: @thibaults-centreon could you please reword to explain that readers have to get in touch with their Centreon sales rep to access to the commercial version please? Idem in FR

**Review by Sims24 — CHANGES_REQUESTED** [review]

File: en/integrations/plugin-packs/procedures/applications-monitoring-alyvix-restapi.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+```bash
+yum install centreon-plugin-centreon-plugin-Applications-Monitoring-Alyvix-Restapi
+```
+
+2. On the Centreon Web interface, install the *Alyvix Server* Centreon Plugin-Pack on the "Configuration > Plugin Packs > Manager" page
+
+<!--Offline IMP License-->
+
+1. Install the Centreon Plugin package on every Centreon poller expected to monitor *Google GSuite* applications:
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/applications-monitoring-alyvix-restapi.md:85]: @thibaults-centreon it's not gsuite here ;)

**Review by Sims24 — CHANGES_REQUESTED** [review]

File: en/integrations/plugin-packs/procedures/applications-monitoring-alyvix-restapi.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+
+3. On the Centreon Web interface, install the *Alyvix Server* Centreon Plugin-Pack on the "Configuration > Plugin Packs > Manager" page
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Configuration
+
+* Log into Centreon and add a new Host through "Configuration > Hosts". 
+* Fill the "Name", "Alias" & "IP Address / DNS" fields according to your Alyvix Server settings
+* Select the *App-Monitoring-Alyvix-Restapi-custom.
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/applications-monitoring-alyvix-restapi.md:105]: Missing closing *

**Review by Sims24 — CHANGES_REQUESTED** [review]

File: en/integrations/plugin-packs/procedures/applications-monitoring-alyvix-restapi.md (hunk 1 above)
Comment (thibaults-centreon) [reply en/integrations/plugin-packs/procedures/applications-monitoring-alyvix-restapi.md:18]: Please suggest

Comment (Sims24) [reply en/integrations/plugin-packs/procedures/applications-monitoring-alyvix-restapi.md:18]: nevermind, I'll push it

**Review by Sims24 — APPROVED** [review]

**Pull request merged**